File: BackEnd/src/Model/Entities/Rol.ts
class Rol {

    private nom: string;

    public constructor(rol: string) {
        this.nom = rol;
    }

    public getNom(): string {
        return this.nom;
    }

    public setNom(nom: string): void {
        this.nom = nom;
    }
}

export = Rol;

File: src/Model/Entities/Rol.java

package hospiolot.model.business.entities;


public class Rol extends Entity {

    private String nom;
    
    public Rol(String rol) {
        this.nom = rol;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    

    
}
